runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4874,6 +4886,9 @@ class INTERPRETER_CLASS
 		J9ClassLoader *classLoader = NULL;
 		PORT_ACCESS_FROM_JAVAVM(_vm);
 		UDATA *bp = buildSpecialStackFrame(REGISTER_ARGS, J9SF_FRAME_TYPE_JNI_NATIVE_METHOD, jitStackFrameFlags(REGISTER_ARGS, J9_STACK_FLAGS_USE_SPECIFIED_CLASS_LOADER), true);
+#if JAVA_SPEC_VERSION >= 19
```

Comment:
Missed fail cases here,  too, so the inc/dec should move deeper in.

Reply:
Updated.